### PR TITLE
Added Sns Support

### PR DESCRIPTION
**Pull request by hasindum**
- **What does the PR do?** 
        Adds SNS CRD support
- **What automated testing exists for this change?** 
        Extended existing test cases by adding sns type 
- **Is there documentation?** no
- **Any dependencies?** no
- [x] **Has the changelog been updated?** Yes

### CODE REVIEW

**Comment by pidah** [comment]
please add tests in test/assets/environments.bitesize

File: pkg/k8_extensions/types.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 // SupportedThirdPartyResources contains all supported TPRs on bitesize
 // cluster.
 var SupportedThirdPartyResources = []string{
-	"mongo", "mysql", "cassandra", "redis", "zookeeper", "kafka", "postgres",
+	"mongo", "mysql", "cassandra", "redis", "zookeeper", "kafka", "postgres", "sns"
```
Comment (gamunu) [inline pkg/k8_extensions/types.go]: add a comma here

**Comment by hasindum** [comment]
@gamunu Updated the version  to 0.0.22

**Comment by gamunu** [comment]
@hasindum Pease followup this PR. And update version accordingly

File: test/assets/environments.bitesize [hunk 2]
```diff
@@ -175,4 +175,12 @@ environments:
...
\ No newline at end of file
+      index_data_size: 512
+- name: environment11
+  namespace: environment-sns
+  services:
+  - type: sns
+    name: sns
+    options:
+      protocol: email
+      endpoint: hasindu.malalaachichige@pearson.com
```
Comment (gamunu) [inline test/assets/environments.bitesize]: Remove email address and add a `email@example.com`

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,14 +9,9 @@ This project adheres to [Semantic Versioning](http://semver.org/). Additionally,
 - **Fixed** for any bug fixes.
 - **Security** for any security changes or fixes for vulnerabilities.
 
-### **[0.0.23] 2019-02-12 [RELEASED]**
```
Comment (gamunu) [inline CHANGELOG.md]: This is removing peters changelog, please fix it.

Comment (hasindum) [reply CHANGELOG.md]: This has been updated.

File: test/assets/environments.bitesize (hunk 2 above)
Comment (hasindum) [reply test/assets/environments.bitesize]: This has been changed to email@example.com

**Comment by mward29** [comment]
I'm not sure it makes much sense to link to a Jira ticket that no one outside Pearson can see in an open source project. 

* **What does the PR do?**
  https://agile-jira.pearson.com/browse/ODY-140
  Adding SNS feature to EO

**Comment by pidah** [comment]
hey @mward29 ! good to see you around :)  yes makes sense not to add an internal reference in this repo.

**Pull request merged**